recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib
+
+./configure \
```

Comment:
@jakirkham Thanks for the heads up. But it may be of use in sorting out the correct `configure` switches.